test/test_datapipe.py

```diff
@@ -343,6 +350,56 @@ def test_demux_mux_datapipe(self):
         self.assertEqual(list(range(10)), list(n))
 
 
+class TestDataFramesPipes(TestCase):
```

Comment:
Is there way to test whether the Capture object is correct?

Reply:
For users they are printable (but with dynamic names), for code, execute and verify results probably the best approach.